ts/src/bingx.ts

```diff
@@ -3029,7 +3030,7 @@ export default class bingx extends Exchange {
             query += 'signature=' + signature;
             headers = {
                 'X-BX-APIKEY': this.apiKey,
-                'X-SOURCE-KEY': 'CCXT',
+                'X-SOURCE-KEY': this.safeString (this.options, 'broker', 'CCXT'),
```

Comment:
@carlosmiei should we expose broker key in bingx?

Reply:
looks ok to me